backends/p4tools/modules/testgen/targets/bmv2/CMakeLists.txt

```diff
@@ -32,8 +32,8 @@ set(
 set(
   TESTGEN_GTEST_SOURCES
   ${TESTGEN_GTEST_SOURCES}
-  ${CMAKE_CURRENT_SOURCE_DIR}/test/backend/ptf.cpp
-  ${CMAKE_CURRENT_SOURCE_DIR}/test/backend/stf.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/test/test_backend/ptf.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/test/test_backend/stf.cpp
```

Comment:
Nit: I think I'd go with just 'backend' here instead of test_backend? But not very important if you have a preference.